gcloud/resource_manager/client.py

```diff
@@ -0,0 +1,215 @@
+                       :class:`NoneType`
+    :param credentials: The OAuth2 Credentials to use for the connection
+                        owned by this client. If not passed (and if no ``http``
+                        object is passed), falls back to the default inferred
+                        from the environment.
+
+    :type http: :class:`httplib2.Http` or class that defines ``request()``.
+    :param http: An optional HTTP object to make requests. If not passed, an
+                 ``http`` object is created that is bound to the
+                 ``credentials`` for the current object.
+    """
+
+    _connection_class = Connection
+
+    @classmethod
+    def from_service_account_json(cls, *args, **kwargs):
+        """Factory to retrieve JSON credentials while creating client.
+
+        The behavior from the parent class is disabled here since the Resource
+        Manager API can only use credentials from a user account, not a
+        service account.
+
+        :raises: :class:`NotImplementedError <exceptions.NotImplementedError>`
+                 always.
+        """
+        raise NotImplementedError('A service account cannot be used with the '
+                                  'Resource Manager API. Only user '
+                                  'credentials can be used.')
+
+    @classmethod
+    def from_service_account_p12(cls, *args, **kwargs):
+        """Factory to retrieve P12 credentials while creating client.
+
+        The behavior from the parent class is disabled here since the Resource
+        Manager API can only use credentials from a user account, not a
+        service account.
+
+        :raises: :class:`NotImplementedError <exceptions.NotImplementedError>`
+                 always.
+        """
+        raise NotImplementedError('A service account cannot be used with the '
+                                  'Resource Manager API. Only user '
+                                  'credentials can be used.')
+
+    def new_project(self, project_id, name=None, labels=None):
+        """Creates a :class:`.Project` bound to the current client.
+
+        Use :meth:`Project.reload() \
+        <gcloud.resource_manager.project.Project.reload>` to retrieve
+        project metadata after creating a :class:`.Project` instance.
+
+        .. note:
+
+            This does not make an API call.
+
+        :type project_id: str
+        :param project_id: The ID for this project.
+
+        :type name: string
+        :param name: The display name of the project.
+
+        :type labels: dict
+        :param labels: A list of labels associated with the project.
+
+        :rtype: :class:`.Project`
+        :returns: A new instance of a :class:`.Project` **without**
+                  any metadata loaded.
+        """
+        return Project(project_id=project_id,
+                       client=self, name=name, labels=labels)
+
+    def fetch_project(self, project_id):
+        """Get an existing project by ID.
```

Comment:
Can we make the docs more clear about this? Fetch an existing project and it's relevant metadata by ID ?